arangod/RocksDBEngine/RocksDBCollection.h

```diff
@@ -25,6 +25,7 @@
 
 #include "Statistics/ServerStatistics.h"
 #include "RocksDBEngine/RocksDBMetaCollection.h"
+#include "RocksDBEngine/RocksDBPrimaryIndex.h"
```

Comment:
It seems that it is not necessary to include this file here. The forward declaration would have been sufficient.

Reply:
Unfortunately it is necessary because I redefine `primaryIndex` with a covariant return type (`RocksDBPrimaryIndex` instead of `Index`), so a forward declaration is not sufficient because the compiler needs to know the definition to check that it is a subtype of `Index`.